telemetry/definitions/commonDefinitions.json

```diff
@@ -414,7 +415,8 @@
         },
         {
             "name": "cloudformation_open",
-            "description": "Open a CloudFormation stack in the stack viewer"
+            "description": "Open a CloudFormation stack in the stack viewer",
+            "metadata": [ { "type": "result", "required": false  } ]
```

Comment:
Other "open" metrics have a required result field. What is the motivation for making these ones optional?

Reply:
I was not certain how this change will affect jetbrains/vscode, so in order to minimize any disruption to them, I made it optional.

Reply:
We don't update automatically, if it's an operation that can fail there is no issue with adding result

Reply:
Done